### PR TITLE
update uses load_entity just like show and destroy

### PR DESCRIPTION
**Pull request by knutsenm**
Passes all unit tests; not spot-checked yet. Suggest we try it in staging after code review.

### CODE REVIEW

File: app/controllers/entities_controller.rb [hunk 1]
```diff
@@ -51,15 +51,11 @@ def destroy_by_entity_type
   end
 
   def update
-    entity = Entity.owned_by(current_system.id).find(id_as_uuid)
-    if entity.root? || entity.owned_by?(current_system)
```
Comment (knutsenm) [inline app/controllers/entities_controller.rb]: Should entity.root? have been current_system.root? If so, the interesting thing is that entity does respond to .root? thanks to acts_as_sane_tree.

**Comment by knutsenm** [comment]
Spot check works. With this patch I can PUT a new client_integration_id and name to some other system's public ministry.

**Comment by twinge** [comment]
:+1:

**Comment by knutsenm** [comment]
Pushed to staging

**Pull request merged**